tests/functional/pyocf/types/volume.py

```diff
@@ -341,6 +344,71 @@ def new_io(
         )
         return Io.from_pointer(io)
 
+    def sync_io(
+        self,
+        queue,
+        address: int,
+        data: Data,
+        direction: IoDir,
+        io_class=0,
+        flags=0,
+        submit_func=Sync.submit,
+    ):
+        assert address % 512 == 0
+        assert data.size % 512 == 0
+
+        io = self.new_io(queue, address, data.size, direction, io_class, flags)
+        io.set_data(data)
+        completion = submit_func(Sync(io))
+
+        assert int(completion.results["err"]) == 0
+
+    def write_sync_4k(
```

Comment:
This method is not used in test. Is it better to remove it or leave it for a future reference?

Reply:
It may come in handy :)